tools/test/perform/iopipe.c

```diff
@@ -107,7 +106,6 @@ synchronize(void)
     status = HDsystem("df >/dev/null");
     HDassert(status >= 0);
 #endif
```

Comment:
Unclear why we're not using sync(2) here instead of system("sync"). Also unclear what the df hack is supposed to be doing.

Reply:
Ancient testcode - not certain either, except that a system sync will flush data from all processes.